study.md

```diff
@@ -56,13 +59,31 @@ If a class in Ruby gets modified while there are existing instances of it, then
 will those instances also have those modifications?
 
 ```md
-<!-- your answer here -->
+yes, you can add code to classes and overwrite existing methods
 ```
 
 ## Iteration
 
 Write an example of iteration in Ruby.
 
 ```ruby
-# your answer here
+a = [7,8,9,10]
+b = a.collect{|x| 10*x} //abs value x * 10
```

Comment:
The syntax `|...|` indicates the parameters for the block, not the absolute value function.